<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionCancel"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionThickness"/>
    <addaction name="actionColor"/>
    <addaction name="actionColorBucket"/>
   </widget>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Shape</string>
    </property>
    <addaction name="actionLigne"/>
    <addaction name="actionCircle"/>
    <addaction name="actionRectangle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuDraw"/>
   <addaction name="menuShape"/>
   <addaction name="menuZoom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionThickness">
   <property name="text">
    <string>Thickness</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionColorBucket">
   <property name="text">
    <string>ColorBucket</string>
   </property>
  </action>
  <action name="actionLigne">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ligne</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
